tests/cases/standalone/common/types/decimal/decimal_small_precision_behavior.result

```diff
@@ -0,0 +1,66 @@
+select '1.023450000001'::DECIMAL(5,4);
+
++------------------------+
+| Utf8("1.023450000001") |
++------------------------+
+| 1.0235                 |
++------------------------+
+
+select '1.234499999'::DECIMAL(4,3);
+
++---------------------+
+| Utf8("1.234499999") |
++---------------------+
+| 1.234               |
++---------------------+
+
+select '1.23499999'::DECIMAL(4,3);
+
++--------------------+
+| Utf8("1.23499999") |
++--------------------+
+| 1.235              |
++--------------------+
+
+select '1.234499999'::DECIMAL(5,4);
+
++---------------------+
+| Utf8("1.234499999") |
++---------------------+
+| 1.2345              |
++---------------------+
+
+-- arrow-rs is a little strange about the conversion behavior of negative numbers.
+-- issue: https://github.com/apache/arrow-datafusion/issues/8326
```

Comment:
We might need to update datafusion for this fix later. Does datafusion update its arrow version?